docs/tasks/run-application/deployment-patch-demo.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
+apiVersion: apps/v1beta2 # for versions before 1.9.0 use apps/v1beta2
```

Comment:
Why this change is needed? We'd better use  (and encourage users to use) `apps/v1` for 1.9+, right?

Reply:
I was experimenting and forgot to change it back to v1. It's back at v1 now.